assets/sass/_pulumi-up.scss

```diff
@@ -0,0 +1,100 @@
+// Neon Animation found on Codepen
+// https://codepen.io/Trinca/pen/NAvpWa
+$pageBackgroundColor: #11151c;
+
+.neon-container {
+  text-align: center;
+  vertical-align: middle;
+}
+
+@mixin neon-sign-defaults($fontSize: 4rem) {
+  font-family: "Vibur";
+  font-size: $fontSize;
+}
+
+@mixin neon-sign-animation($light, $medium, $dark, $pulseTime, $name) {
+    animation: $name $pulseTime linear infinite;
+    -moz-animation: $name $pulseTime linear infinite;
+    -webkit-animation: $name $pulseTime linear infinite;
+    -o-animation: $name $pulseTime linear infinite;
+
+    @keyframes #{$name} {
+      0%,
+      100% {
+        text-shadow: 0 -40px 100px, 0 0 2px, 0 0 1em $medium, 0 0 0.5em $medium, 0 0 0.1em $medium, 0 10px 3px #000;
+        color: $light;
+      }
+      50% {
+        text-shadow: 0 -40px 100px, 0 0 2px, 0 0 1em $dark, 0 0 0.5em $dark, 0 0 0.1em $dark, 0 10px 3px #000;
+        color: $medium;
+      }
+    }
+}
+
+@mixin neon-sign-purple($fontSize, $name: "neon-sign-purple-pulse", $animate: true) {
+  @include neon-sign-defaults($fontSize);
+  @if $animate {
+    @include neon-sign-animation(theme('colors.purple.300'), theme('colors.purple.600'), theme('colors.purple.800'), 4s, $name);
+  }
+
+  color: theme('colors.purple.400');
```

Comment:
We're repeating these `theme()` lookups a lot -- it'd be nice if we could just do the lookups once and pull the values into a few local variables, and just reference them that way.

Reply:
👍